simple_ehm-runnable.py

```diff
@@ -266,8 +281,17 @@ def cut_and_merge(out_filename):
     minute = str(datetime.datetime.now().minute)
     secs = str(datetime.datetime.now().second)
     
-    out_filename += "_" + hour + "-" + minute + "-" + secs + video_path[-4:]
+    if(not args.name):
+    	out_filename += "_" + hour + "-" + minute + "-" + secs 
     
+    else:
+	    for j, i in enumerate(args.name):
+		    out_filename += i
+		    if (len(args.name) != 1 and j != len(args.name)-1):
+		        out_filename+="_"
+
+    out_filename += video_path[-4:] 
+
```

Comment:
Ok so if I'm not mistaken you're expecting the user to enter a suffix or a "python's stlye array definition" suffix by the user, like so:
`["suffix", "name", "here"]`
And then you concatenate the string to `suffix_name_here`. Is that correct?
In that case I'd suggest you just let the user specify a quoted phrase like `"suffix name here"` as a normal string and then translate/substitute all the spaces with "_" and it should be just a one liner.

Reply:
Ok, ho capito.
È una soluzione intressante ma preferisco costringere l'utente a chiudere la frase tra doppi apici (pratica abbastanza comune in questi casi) che complicare il codice. Anche se si tratta di poche righe, ti garantisco che mi ci sono voluti alcuni minuti per capire cosa facesse, e questo script è già abbastanza incasinato di suo.